src/services/state.service.ts

```diff
@@ -33,4 +38,58 @@ export class StateService
       this.reconcileOptions(options, await this.defaultOnDiskLocalOptions())
     );
   }
+
+  async getEncryptedCiphers(options?: StorageOptions): Promise<{ [id: string]: CipherData }> {
```

Comment:
I wonder if we should add the `override` key here to make it obvious we're overriding stuff. But to be fair I don't believe we do it in other places. (We should turn on the --no-implicit-override flag in TS)